contents/product-engineers/mobile-app-metrics.md

```diff
@@ -0,0 +1,579 @@
+---
+date: 2024-09-16
+title: 25 best mobile app metrics to track
+author:
+  - lior-neu-ner
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - Product metrics
+crosspost:
+  - Blog
+---
+
+## Sign up and onboarding metrics
+
+### 1. App downloads
+
+**What it is:** The number of times your app has been downloaded.
+
+**Why it's useful:** Provides a baseline metric for user acquisition and helps measure the effectiveness of marketing campaigns.
+
+**Questions to ask:**
+- How does the number of downloads vary across different platforms (iOS, Android, etc.)?
+- Is there a correlation between the number of downloads and user engagement or retention?
+- How do different marketing channels impact the number of downloads?
+
+**Insight type:** [Trend](/docs/product-analytics/trends/overview)
+
+<ProductScreenshot
+    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/v1726494646/posthog.com/contents/Screenshot_2024-09-16_at_2.49.24_PM.png"
+    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/v1726494646/posthog.com/contents/Screenshot_2024-09-16_at_2.49.43_PM.png"
+    classes="rounded"
+    alt="Number of app downloads"
+/>
+
+### 2. App bounce rate
+
+**What it is:** The percentage of users who view your home screen but don't view a second screen.
+
+**Why it's useful:** Enables you to learn if your welcome messaging is resonating with news users.
+
+**Good benchmark:** Below 9% is considered good – [Source](https://contentsquare.com/blog/app-metrics/)
+
+**Questions to ask:**
+- How does the bounce rate correlate with downloads from different marketing channels?
+- Are users in specific countries more likely to bounce? How does localization correlate with bounce rate?
+- How does the bounce rate vary across different devices and app versions?
+
+**Insight type:** [Trend](/docs/product-analytics/trends/overview)
```

Comment:
Case in point, in this example you presumably have to do a formula to get this metric? We should explain how to do this as succinctly as possible.